simpleclient_servlet/src/main/java/io/prometheus/client/filter/MetricsFilter.java

```diff
@@ -0,0 +1,128 @@
+package io.prometheus.client.filter;
+
+import io.prometheus.client.Histogram;
+import org.apache.commons.lang3.StringUtils;
```

Comment:
If we can avoid the dep then it can stay here, we'll just need to be clear in the readme where this comes from.

The other consideration is that we'd want a fairly new jetty for the servlet in order to fix whatever bugs/issues there are, however we'd want to have as old a jetty as practical to ensure we're not unintentionally using new features that users stuck on an old version wouldn't have. Try and find a good old stable version I guess.